src/exes/moc.ml

```diff
@@ -106,6 +114,10 @@ let argspec = [
     (fun () -> Flags.sanity := true),
   " enable sanity checking in the RTS and generated code";
 
+  "--stable-types",
+  Arg.Unit (fun () -> stable_types := true),
+  " emit signature of stable types to .sig file";
```

Comment:
Can we use a more specific file extension? How about `.stable`, or `.most`?

Reply:
I liked .moi ;->

Reply:
Changed  to .most in full PR #2887